package.json

```diff
@@ -59,19 +59,15 @@
     "yorkie": "^2.0.0"
   },
   "gitHooks": {
-    "pre-commit": "lint-staged",
+    "pre-commit": "lint-staged --concurrent false",
     "commit-msg": "node scripts/verifyCommit.js"
   },
   "lint-staged": {
-    "*.js": [
-      "prettier --write"
+    "*": [
+      "prettier --write ."
     ],
     "*.ts": [
-      "eslint",
-      "prettier --parser=typescript --write"
-    ],
-    "*.html": [
-      "prettier --write"
+      "eslint --ext .ts packages/*/src/**/*.ts"
```

Comment:
`lint-staged` will pass the staged files as arguments to these commands, so the last argument here is redundant.

Reply:
I assumed that `lint-staged` is doing that 🙁 
The problem with that is, that sometimes files get generated or rebuild out of the commit stage resulting in ignored files.
I would like to always format and lint the whole code/repo when someone tries to create a commit so there are no future inconsistencies.

Can I tell `lint-staged` to not pass the arguments? Or is there another way?

Reply:
But that's the whole point of `lint-staged` - to only lint staged files. It helps speed things up and avoid introducing unrelated code changes in a commit or PR.
As I tested, running `prettier` on the entire repo takes quite some time on my machine.

I think ideally, what we need is to:
* run `yarn format && yarn lint` when
  * formatting rules changed
  * `prettier` is updated
  * new files are generated (I'm not sure in which circumstances will there be files generated out of the commit stage, though. Do you have any examples?)
* lint only staged files when
  * regular source files changed

Maybe there's better alternatives than `lint-staged` to cover the first scenario?

Reply:
>   * new files are generated (I'm not sure in which circumstances will there be files generated out of the commit stage, though. Do you have any examples?)

I think package.json files in create-app sometimes get ignored, cause somehow the eol is removed but added when running prettier manually?! Need to look if the `package.json`s are generated.

> * lint only staged files when
>   
>   * regular source files changed

Until now only the vite src files get formatted, but not files like create-app, yml, md or playground files (I fixed that now in another branch #4154)
Also merge commits and dependency updates seem to ignore the lint-staged execution?!

But yes, I will look into this PR later and try another solution that only takes committed files into account